Set3/20/main.cc

```diff
@@ -0,0 +1,35 @@
+#include "main.ih"
+
+int main(int argc, char *argv[])
+{
+    size_t count = 0;
+    
+    if (argc == 1)
+    {
+        usage();
+        return 0;
+    }
+    
+    char const selector = argv[1][1];
```

Comment:
```suggestion 
    if (argc == 1)
    {
        usage();
        return 0;
    }
    
    char const selector = argv[1][1];
    size_t count = 0;
```
`count` should be defined closer to where it's actually used.